src/main/java/name/valery1707/kaitai/KaitaiMojo.java

```diff
@@ -27,8 +27,8 @@
  * @see <a href="http://doc.kaitai.io/user_guide.html#_invocation">Kaitai compiler documentation</a>
  */
 @Mojo(
-	name = "generate"
-	, defaultPhase = LifecyclePhase.GENERATE_SOURCES
+		name = "generate"
+		, defaultPhase = LifecyclePhase.GENERATE_SOURCES
```

Comment:
Please revert the indentation changes
```suggestion
	name = "generate"
	, defaultPhase = LifecyclePhase.GENERATE_SOURCES
```